base_import_match/models/base_import.py

```diff
@@ -88,7 +89,7 @@ def _compute_name(self):
                 " + ".join(
                     s.field_ids.mapped(
                         lambda r: (
-                            str(r.field_id.name) +
+                            unicode(r.field_id.name) +
```

Comment:
I think you don't get the point. I have to use `unicode` because otherwise if `r.field_id.name is False` this would blow.